src/custom-hooks/getVideosData.jsx

```diff
@@ -0,0 +1,19 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+const useVideosDataHook = () => {
```

Comment:
You are performing same logic in `useVideosDataHook` and `useCategoriesDataHook`. So you can make a common `useAxios` hook which can work in both cases when provided with different parameters.